### PR TITLE
[CS2113T-F14-4]-lackmoo-B-FixedDurationTasks

### PR DESCRIPTION
**Pull request by lackmoo**

### CODE REVIEW

File: src/main/java/duke/commands/AddCommand.java [hunk 1]
```diff
@@ -34,6 +35,8 @@ public AddCommand(String[] components, String fullCommand) throws DukeException
             throw new InputException("☹ OOPS!!! The description of a deadline cannot be empty.");
         } else if (this.type.equals("event") && components[1].equals("/by")) {
             throw new InputException("☹ OOPS!!! The description of an event cannot be empty.");
+        } else if (this.type.equals("event") && components[1].equals("/needs")) {
```
Comment (rssujay) [inline src/main/java/duke/commands/AddCommand.java]: Please fix the bug here.

File: src/test/java/FixedTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import duke.tasks.Task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class FixedTest {
+
+    @org.junit.jupiter.api.Test
+    void testToString() {
+        Task t = new Fixed("read sales report", "2 hours");
```
Comment (rssujay) [inline src/test/java/FixedTest.java]: 1) Don't use single character variable names - something like Task task would be better.
2) Follow unit test naming conventions. 

image

File: src/main/java/duke/commands/AddCommand.java (hunk 1 above)
Comment (lackmoo) [reply src/main/java/duke/commands/AddCommand.java]: Thanks for spotting this trivial mistake!

File: src/test/java/FixedTest.java (hunk 2 above)
Comment (lackmoo) [reply src/test/java/FixedTest.java]: Alright, will stick to the naming conventions!

**Pull request merged**